advocacy_docs/pg_extensions/wait_states/installing.mdx

```diff
@@ -23,13 +52,51 @@ To install EDB Wait States on a Linux system:
   CREATE EXTENSION edb_wait_states;
   ```
 
-1. To terminate the EDB Wait States worker, remove `$libdir/edb_wait_states` from the `shared_preload_libraries` parameter, and restart the database server.
+## On Debian and Ubuntu
 
-## On Windows
+To install EDB Wait States on Debian or Ubuntu:
+
+1.  Enter the following command:
+
+  ```shell
+  sudo apt-get -y install <install-command>
```

Comment:
idem above on the usage of `install-command`

Reply:
Revised to use "install-package-name"